account_reconcile_oca/tests/test_bank_account_reconcile.py

```diff
@@ -10,6 +10,16 @@ class TestReconciliationWidget(TestAccountReconciliationCommon):
     @classmethod
     def setUpClass(cls, chart_template_ref=None):
         super().setUpClass(chart_template_ref=chart_template_ref)
+        # Auto-disable reconciliation model created automatically with
+        # generate_account_reconcile_model() to avoid side effects in tests
+        cls.invoice_matching_model = cls.env["account.reconcile.model"].search(
```

Comment:
If it can be more than one, the variable name is not correct

Reply:
I have changed the variable name to be consistent. As a curiosity to indicate that there will be one model per company created in https://github.com/odoo/odoo/blob/dbd2f89b4fe88488d3b02e475fae6ac8878694ae/addons/account/models/chart_template.py#L1085.